jsonschema-core.xml

```diff
@@ -419,13 +419,25 @@
                     <t>
                         A JSON Schema resource is a schema which is
                         <xref target="RFC6596">canonically</xref> identified by an
-                        <xref target="RFC3986">absolute URI</xref>.
+                        <xref target="RFC3986">absolute URI</xref>.  Schema resources MAY
+                        also be identified by URIs, including URIs with fragments,
+                        if the resulting secondary resource (as defined by
+                        <xref target="RFC3986">section 3.5 of RFC 3986</xref>) is identical
+                        to the primary resource.  This can occur with the empty fragment,
+                        or when one schema resource is embedded in another.  Any such URIs
```

Comment:
This can also happen if there is a `$dynamicAnchor` and/or `$anchor` at this location, but I'm not sure if it's worth mentioning that here.

Reply:
Let's break this into a new issue if you feel strongly enough about it =]